src_testbed/testbed.rs

```diff
@@ -1349,67 +1267,62 @@ impl State for Testbed {
                     {
                         //                        println!("Step");
                         self.physx.as_mut().unwrap().step(
-                            &mut self.physics.pipeline.counters,
-                            &self.physics.integration_parameters,
+                            &mut physics.pipeline.counters,
+                            &physics.integration_parameters,
                         );
                         self.physx
                             .as_mut()
                             .unwrap()
-                            .sync(&mut self.physics.bodies, &mut self.physics.colliders);
+                            .sync(&mut physics.bodies, &mut physics.colliders);
                     }
                 }
 
                 #[cfg(feature = "other-backends")]
                 {
                     if self.state.selected_backend == NPHYSICS_BACKEND {
                         self.nphysics.as_mut().unwrap().step(
-                            &mut self.physics.pipeline.counters,
-                            &self.physics.integration_parameters,
+                            &mut physics.pipeline.counters,
+                            &physics.integration_parameters,
                         );
                         self.nphysics
                             .as_mut()
                             .unwrap()
-                            .sync(&mut self.physics.bodies, &mut self.physics.colliders);
+                            .sync(&mut physics.bodies, &mut physics.colliders);
                     }
                 }
 
-                for f in &mut self.callbacks {
-                    f(
-                        window,
-                        &mut self.physics,
-                        &self.events,
-                        &mut self.graphics,
-                        self.time,
-                    )
-                }
-
                 for plugin in &mut self.plugins {
-                    plugin.run_callbacks(window, &mut self.physics, self.time)
+                    {
+                        plugin.run_callbacks(
+                            window,
+                            &mut self.harness.physics,
+                            &self.harness.state,
+                        );
+                    }
```

Comment:
```suggestion
                    plugin.run_callbacks(
                        window,
                        &mut self.harness.physics,
                        &self.harness.state,
                    );
```
Looks like there are some useless braces?